cmd/id-exporter/main.go

```diff
@@ -48,18 +86,44 @@ func (c idExporter) findIDs() ([]id, error) {
 		c.log.AuditErrf("Error finding IDs: %s", err)
 		return nil, err
 	}
+	return holder, nil
+}
 
-	return idsList, nil
+// Find all registration IDs with unexpired certificates and gather an
+// example hostname.
+func (c idExporter) findIDsWithExampleHostnames() (idExporterResults, error) {
+	var holder idExporterResults
+	_, err := c.dbMap.Select(
```

Comment:
Did you want that comment draft, or shall we just leave it in the Github history?